cms/djangoapps/contentstore/views/helpers.py

```diff
@@ -25,6 +25,14 @@
 
 __all__ = ['edge', 'event', 'landing']
 
+GRADER_TYPES = {
```

Comment:
Maybe a comment - this is not a complete list of grader types because they are user-changeable, but rather a list of common types.